testing/integration-tests/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "integration_tests"
```

Comment:
Package names are supposed to be global, so I'd name it "jormungandr-integration-tests". It's more conventional to use dashes in package names.